src/components/common/nextorback/NextOrBack.css

```diff
@@ -6,13 +6,21 @@
   width: 40px;
 }
 
+.next-text{
+  position: absolute;
+  white-space: nowrap;
+  font-size: 10px;
+  bottom: 50px;
+}
+
 .next-symbol {
   position: absolute;
   bottom: 0px;
 }
 
 .next-symbol:hover{
   cursor: pointer;
+  transition: linear 0.2s;
```

Comment:
This won't do anything so it should be removed. I found this on stack overflow 
"No, there's no way. The title attribute is implemented in a browser dependent fashion."

Basically, browsers create their own rules about how fast a tool tip comes up and there's nothing you can do about it. If you wanted to create a tooltip you would have to do it from scratch.